DigitalLearningSolutions.Data/DataServices/UserDataService/DelegateUserCardDataService.cs

```diff
@@ -7,41 +7,43 @@
 
     public partial class UserDataService
     {
+        private const string DelegateUserSelectQuery = @"cd.CandidateID AS Id,
+                                                        cd.CandidateNumber,
+                                                        ct.CentreName,
+                                                        cd.CentreID,
+                                                        cd.DateRegistered,
+                                                        ct.Active AS CentreActive,
+                                                        cd.EmailAddress,
+                                                        cd.FirstName,
+                                                        cd.LastName,
+                                                        cd.Password,
+                                                        cd.Approved,
+                                                        cd.Answer1,
+                                                        cd.Answer2,
+                                                        cd.Answer3,
+                                                        cd.Answer4,
+                                                        cd.Answer5,
+                                                        cd.Answer6,
+                                                        cd.JobGroupId,
+                                                        jg.JobGroupName,
+                                                        cd.SelfReg,
+                                                        cd.ExternalReg,
+                                                        cd.Active,
+                                                        (SELECT AdminID
+                                                        FROM AdminUsers au
+                                                            WHERE (au.Email = cd.EmailAddress
+                                                            OR au.Email = cd.AliasID)
+                                                        AND au.Password = cd.Password
+                                                            AND au.CentreID = cd.CentreID
+                                                            AND au.Email != ''
+                                                        ) AS AdminID,
+                                                            cd.AliasID ";

```

Comment:
NitPik - Indentation is a little out on this line.

Reply:
I ran code cleanup